HISTORY.md

```diff
@@ -1,5 +1,28 @@
 # Release Notes
 
+## 0.14.0 - 2022-03-17
+
+This release updates the sampling API and splits the existing functionality into three methods - `sample`, `sample_conditions`,
+and `sample_remaining_columns`. We also add support for a progress bar when sampling with a batch size, and writing
```

Comment:
1. Sampling by batches in itself is also a new functionality (as opposed to just the progress bar)
2. We are also adding the ability to sample deterministically